app/components/job/action/job-commands.ts

```diff
@@ -189,4 +197,12 @@ export class JobCommands extends EntityCommands<Job> {
             return Observable.fromPromise(this.fs.saveFile(localPath, content));
```

Comment:
`Observable.fromPromise` change to `from` in genera change all Observable. to the rxjs 6 version